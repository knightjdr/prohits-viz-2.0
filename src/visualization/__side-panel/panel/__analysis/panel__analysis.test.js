import React from 'react';
import { shallow } from 'enzyme';

import Analysis from './panel__analysis';

describe('Analysis panel', () => {
  it('should render', () => {
    const wrapper = shallow(
      <Analysis />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
