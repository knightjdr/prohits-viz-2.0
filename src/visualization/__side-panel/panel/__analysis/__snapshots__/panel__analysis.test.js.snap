// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Analysis panel should render 1`] = `
<div
  className="panel"
>
  <Connect(SelectionContainer) />
  <div
    className="panel__border"
  />
  <AnalysisOptionsContainer />
</div>
`;
